src/main/resources/validationrules/iac/iac-caseworker-mapping.drl

```diff
@@ -47,9 +48,10 @@ when
   $f:  FeatureFlag(status && flagName == FeatureFlagEnum.IAC_1_0.getValue())
   $up: CaseWorkerAccessProfile(roleId == "2", serviceCode == "BFA1", !suspended)
 then
-     Map<String,JsonNode> attr = new HashMap<>();
-     attr.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("IA"));
-     attr.put("primaryLocation", JacksonUtils.convertObjectIntoJsonNode($up.getPrimaryLocationId()));
+     Map<String,JsonNode> attribute = new HashMap<>();
+     attribute.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("IA"));
+     attribute.put("primaryLocation", JacksonUtils.convertObjectIntoJsonNode($up.getPrimaryLocationId()));
+     attribute.put("substantive", JacksonUtils.convertObjectIntoJsonNode("Y"));
```

Comment:
we don't need to make any changes in the IAC_1_0 rules. All changes will go under IAC_1_1 flag rules.